src/format_string.h

```diff
@@ -34,8 +37,12 @@ class FormatString {
    */
   FormatString() = default;
 
-  FormatString(const char *s) : fmt_(s){};
-  FormatString(std::string &s) : fmt_(s){};
+  FormatString(const char *s) : fmt_(s)
+  {
```

Comment:
why this change?

Reply:
dunno. clang-format really wants to do this and it won't let me leave it unchanged :P